gifted_build/src/server/routes/test.js

```diff
@@ -1,8 +1,31 @@
+const knex = require('knex')({
+    client: 'pg',
+    connection: {
```

Comment:
rather than hardcoding the configs into the route, I would recommend elevating the knex connection into the server config file and then reading the elements from a database.ini file stored at root, so we can pass the connection to the multiple roots and we the code will be relatively the same when switching from dev to prod. we can set a prod_database.ini file too for production configs, but don't worry about for now

Reply:
I put the knex connection stuff in a `database.js` file. I left it as hardcoded for now because there is no production configuration.